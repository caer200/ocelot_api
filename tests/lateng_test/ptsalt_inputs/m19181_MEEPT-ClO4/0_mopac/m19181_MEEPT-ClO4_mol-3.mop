CHARGES

 
H	0.935681 +1 12.458388 +1 15.632405 +1
C	1.058746 +1 11.934157 +1 16.415041 +1
C	1.426521 +1 12.566881 +1 17.624875 +1
C	0.876948 +1 10.572995 +1 16.352717 +1
C	1.599890 +1 11.734109 +1 18.758685 +1
N	1.591602 +1 13.948960 +1 17.684393 +1
H	0.619450 +1 10.173926 +1 15.529881 +1
C	1.064318 +1 9.764310 +1 17.480413 +1
C	1.420252 +1 10.347085 +1 18.672986 +1
S	1.960352 +1 12.321321 +1 20.341569 +1
C	2.008901 +1 14.644183 +1 18.817383 +1
C	1.192482 +1 14.733051 +1 16.481409 +1
H	0.947285 +1 8.823350 +1 17.421745 +1
H	1.545393 +1 9.806323 +1 19.443890 +1
C	2.175375 +1 14.009304 +1 20.073959 +1
C	2.283269 +1 16.030699 +1 18.747987 +1
H	0.409700 +1 14.297865 +1 16.059996 +1
H	0.916386 +1 15.639909 +1 16.766629 +1
C	2.297897 +1 14.857922 +1 15.451977 +1
C	2.523577 +1 14.753208 +1 21.213331 +1
H	2.206273 +1 16.484762 +1 17.917086 +1
C	2.660796 +1 16.725796 +1 19.871119 +1
H	1.943774 +1 15.268718 +1 14.623671 +1
H	2.654220 +1 13.961891 +1 15.227884 +1
O	3.326552 +1 15.666861 +1 15.991925 +1
H	2.591859 +1 14.319974 +1 22.055952 +1
C	2.766670 +1 16.098650 +1 21.119763 +1
H	2.854553 +1 17.653115 +1 19.799548 +1
C	4.215480 +1 16.188532 +1 15.005089 +1
H	3.003829 +1 16.599720 +1 21.891204 +1
H	4.876375 +1 15.498342 +1 14.746091 +1
H	3.707380 +1 16.452245 +1 14.197338 +1
C	4.916900 +1 17.378296 +1 15.578795 +1
H	4.256145 +1 17.997760 +1 15.978092 +1
H	5.401123 +1 17.860642 +1 14.862341 +1
O	5.838287 +1 16.943592 +1 16.577398 +1
C	6.225705 +1 18.021543 +1 17.406343 +1
H	6.552578 +1 18.759767 +1 16.850587 +1
H	5.455390 +1 18.323744 +1 17.930926 +1
H	6.938617 +1 17.728253 +1 18.011628 +1
