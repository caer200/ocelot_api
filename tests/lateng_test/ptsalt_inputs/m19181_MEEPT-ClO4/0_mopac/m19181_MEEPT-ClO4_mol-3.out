 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:09:21 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          0.93568100  *  12.45838800  *  15.63240500  *
     2       C          1.05874600  *  11.93415700  *  16.41504100  *
     3       C          1.42652100  *  12.56688100  *  17.62487500  *
     4       C          0.87694800  *  10.57299500  *  16.35271700  *
     5       C          1.59989000  *  11.73410900  *  18.75868500  *
     6       N          1.59160200  *  13.94896000  *  17.68439300  *
     7       H          0.61945000  *  10.17392600  *  15.52988100  *
     8       C          1.06431800  *   9.76431000  *  17.48041300  *
     9       C          1.42025200  *  10.34708500  *  18.67298600  *
    10       S          1.96035200  *  12.32132100  *  20.34156900  *
    11       C          2.00890100  *  14.64418300  *  18.81738300  *
    12       C          1.19248200  *  14.73305100  *  16.48140900  *
    13       H          0.94728500  *   8.82335000  *  17.42174500  *
    14       H          1.54539300  *   9.80632300  *  19.44389000  *
    15       C          2.17537500  *  14.00930400  *  20.07395900  *
    16       C          2.28326900  *  16.03069900  *  18.74798700  *
    17       H          0.40970000  *  14.29786500  *  16.05999600  *
    18       H          0.91638600  *  15.63990900  *  16.76662900  *
    19       C          2.29789700  *  14.85792200  *  15.45197700  *
    20       C          2.52357700  *  14.75320800  *  21.21333100  *
    21       H          2.20627300  *  16.48476200  *  17.91708600  *
    22       C          2.66079600  *  16.72579600  *  19.87111900  *
    23       H          1.94377400  *  15.26871800  *  14.62367100  *
    24       H          2.65422000  *  13.96189100  *  15.22788400  *
    25       O          3.32655200  *  15.66686100  *  15.99192500  *
    26       H          2.59185900  *  14.31997400  *  22.05595200  *
    27       C          2.76667000  *  16.09865000  *  21.11976300  *
    28       H          2.85455300  *  17.65311500  *  19.79954800  *
    29       C          4.21548000  *  16.18853200  *  15.00508900  *
    30       H          3.00382900  *  16.59972000  *  21.89120400  *
    31       H          4.87637500  *  15.49834200  *  14.74609100  *
    32       H          3.70738000  *  16.45224500  *  14.19733800  *
    33       C          4.91690000  *  17.37829600  *  15.57879500  *
    34       H          4.25614500  *  17.99776000  *  15.97809200  *
    35       H          5.40112300  *  17.86064200  *  14.86234100  *
    36       O          5.83828700  *  16.94359200  *  16.57739800  *
    37       C          6.22570500  *  18.02154300  *  17.40634300  *
    38       H          6.55257800  *  18.75976700  *  16.85058700  *
    39       H          5.45539000  *  18.32374400  *  17.93092600  *
    40       H          6.93861700  *  17.72825300  *  18.01162800  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          0.9357     12.4584     15.6324
     2         C          1.0587     11.9342     16.4150
     3         C          1.4265     12.5669     17.6249
     4         C          0.8769     10.5730     16.3527
     5         C          1.5999     11.7341     18.7587
     6         N          1.5916     13.9490     17.6844
     7         H          0.6195     10.1739     15.5299
     8         C          1.0643      9.7643     17.4804
     9         C          1.4203     10.3471     18.6730
    10         S          1.9604     12.3213     20.3416
    11         C          2.0089     14.6442     18.8174
    12         C          1.1925     14.7331     16.4814
    13         H          0.9473      8.8233     17.4217
    14         H          1.5454      9.8063     19.4439
    15         C          2.1754     14.0093     20.0740
    16         C          2.2833     16.0307     18.7480
    17         H          0.4097     14.2979     16.0600
    18         H          0.9164     15.6399     16.7666
    19         C          2.2979     14.8579     15.4520
    20         C          2.5236     14.7532     21.2133
    21         H          2.2063     16.4848     17.9171
    22         C          2.6608     16.7258     19.8711
    23         H          1.9438     15.2687     14.6237
    24         H          2.6542     13.9619     15.2279
    25         O          3.3266     15.6669     15.9919
    26         H          2.5919     14.3200     22.0560
    27         C          2.7667     16.0986     21.1198
    28         H          2.8546     17.6531     19.7995
    29         C          4.2155     16.1885     15.0051
    30         H          3.0038     16.5997     21.8912
    31         H          4.8764     15.4983     14.7461
    32         H          3.7074     16.4522     14.1973
    33         C          4.9169     17.3783     15.5788
    34         H          4.2561     17.9978     15.9781
    35         H          5.4011     17.8606     14.8623
    36         O          5.8383     16.9436     16.5774
    37         C          6.2257     18.0215     17.4063
    38         H          6.5526     18.7598     16.8506
    39         H          5.4554     18.3237     17.9309
    40         H          6.9386     17.7283     18.0116

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
